test/t8n/t8n.cpp

```diff
@@ -128,6 +136,16 @@ int main(int argc, const char* argv[])
                 }
             }
 
+            if (block_reward.has_value())
+            {
+                if (*block_reward == 0)
+                    state.touch(block.coinbase);
```

Comment:
This should handle both cases.

```suggestion
                    state.touch(block.coinbase).balance += *block_reward;
```

Reply:
Adding 0 does not change so much :)